lib/FusionInventory/Agent/Task/Inventory/Win32/Crontasks.pm

```diff
@@ -0,0 +1,239 @@
+            }
+            if ($trigger->find('ScheduleByMonth')) {
+                if ($trigger->find('ScheduleByMonth/DaysOfMonth')) {
+                    my @wdays;
+                    my $daysOfMonth = $trigger->findnodes('ScheduleByMonth/DaysOfMonth');
+                    foreach my $day ($daysOfMonth->get_nodelist) {
+                        push(@wdays, $day->string_value);
+                    }
+                    $executionDay = join(",", @wdays);
+                }
+                if ($trigger->find('ScheduleByMonth/Months')) {
+                    my @months;
+                    my $byMonths = $trigger->findnodes('ScheduleByMonth/Months');
+                    if ($trigger->find('ScheduleByMonth/Months/January')) {
+                        push(@months, 1);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/February')) {
+                        push(@months, 2);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/March')) {
+                        push(@months, 3);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/April')) {
+                        push(@months, 4);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/May')) {
+                        push(@months, 5);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/June')) {
+                        push(@months, 6);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/July')) {
+                        push(@months, 7);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/August')) {
+                        push(@months, 8);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/September')) {
+                        push(@months, 9);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/October')) {
+                        push(@months, 10);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/November')) {
+                        push(@months, 11);
+                    }
+                    if ($trigger->find('ScheduleByMonth/Months/December')) {
+                        push(@months, 12);
+                    }
+                    $executionMonth = join(",", @months);
+                }
+            }
+        }
+        # Replace */1 in * (more simple to read)
+        if ($executionMonth eq '*/1') {
+            $executionMonth = '*';
+        }
+        if ($executionDay eq '*/1') {
+            $executionDay = '*';
+        }
+        if ($executionHour eq '*/1') {
+            $executionHour = '*';
+        }
+        if ($executionMinute eq '*/1') {
+            $executionMinute = '*';
+        }
+        if ($executionWeekday eq '*/1') {
+            $executionWeekday = '*';
+        }
```

Comment:
You still assume vars are defined and there should be a cleaner way todo than using 5 times the same kind of `if`. I even don't see how `$executionMonth` & `$executionWeekday` could be set to '*/1'. Indeed day, hour and minute cases should be handled earlier, l.94, l.101 & l.108.
For example, l.94, try this:
```
$executionMinute = '*/'.$2;
```
could be replace by:
```
$executionMinute = $2 && $2 eq '1' ? '*' : '*/'.$2;
```